tests/common/snappi_tests/snappi_test_params.py

```diff
@@ -22,7 +22,21 @@ def __init__(self):
                                              then pcap is on the tgen egress port
             base_flow_config (dict): base flow configuration
             test_tx_frames (list): number of test frames transmitted for priorities to test ex. [2000, 3000]
-                for priorities 3 and 4
+                                    for priorities 3 and 4
+            duthost1 (obj): DUT 1 in case of multi-dut testbed environment
+            rx_port (dict): Contains the details of the rx port
```

Comment:
This is already in the `base_flow_config` parameter of this object - see `base_flow_config["rx_port_config"]` or `base_flow_config["tx_port_config"]`

Reply:
You can look at the `setup_base_flow_config()` function in `traffic_generation.py`

Reply:
fixed this